web/yo/app/scripts/services/mainutils.js

```diff
@@ -227,6 +227,28 @@ angular.module('oncokbApp')
         }
 
         /**
+         * Check whether searched mainType in cancerTypes
+         * @param {array} cancerTypes array of cancer types
+         * @param {string} mainType searched main type
+         * @return {boolean} whether exist
+         */
+        function containMainType(cancerTypes, mainType) {
+            if (!cancerTypes || !mainType) {
+                return null;
+            }
+            var contain = false;
+            mainType = mainType.toLowerCase().trim();
+            cancerTypes.asArray().forEach(function(cancerType) {
+                // console.log(cancerType.cancerType.text);
+                // console.log(cancerType.cancerType.text.indexOf(mainType));
+                if (cancerType.cancerType && cancerType.cancerType.text.toLowerCase().indexOf(mainType) !== -1) {
+                    contain = true;
```

Comment:
should we use "return true" instead? There is no need to continue the loop once contain is set to true, right?